migrate.sh

```diff
@@ -47,20 +47,23 @@ psql $DB_URI -f drop_indices.sql
 
 
 # Run migration
+cd -
```

Comment:
Doesn't this go to the "previous" directory? I think if you want to assume the parent folder of this script file you have to do `script_path=$(dirname "$0")` and use that (maybe it's a good idea in general to build our paths around this assumption/point?)